mobile/pubspec.yaml

```diff
@@ -174,10 +174,11 @@ dependencies:
   widgets_to_image: ^0.0.2
 
 dependency_overrides:
-  # current fork of tfite_flutter_helper depends on ffi: ^1.x.x
-  # but we need ffi: ^2.0.1 for newer packages. The original tfite_flutter_helper
-  #
-  ffi: ^2.1.0
+  connectivity_plus: ^4.0.0
+  
+  #Remove this after  upgrading to flutter v3.19x
```

Comment:
This should be removed then as we are already bumping flutter to v3.19.x?

Reply:
Seems like `flutter_sodium` depends on ^1.0.0. Once we remove dependency from that, we should be able to remove the override here.